torch/_dynamo/variables/builtin.py

```diff
@@ -607,7 +607,7 @@ def _call_min_max_binary(self, tx, a, b):
                 a = variables.TorchVariable(torch.tensor).call_function(tx, [a], {})
 
             # Dynamic input does not get resolved, rather, gets stored as call_function
-            if isinstance(a, SymNodeVariable):
+            if isinstance(a, SymNodeVariable) or isinstance(b, SymNodeVariable):
```

Comment:
This is numpy scalar shenanigans. If I have max(numpy_scalar, sym_int), I MUST NOT run the Tensor operation, because torch.maximum doesn't accept int argument. The directionality here goes both ways, as is being adjusted here.

Reply:
But you can run `clamp` as below, it accepts int args?

Reply:
Oh yeah, I could alternately reroute this to run clamp instead of maximum/minimum. Uhhh I'm not sure which is better.

Reply:
Given our new design in https://docs.google.com/document/d/1INSCdYu1PxXcr43HrD82OudeEuS-qxQe1yZmLg2wy6A/edit#  because numpy_scalar is repped as a 0d tensor this means that we SHOULD tensorify here. I'll do this update.